griptape/tools/date_time/tool.py

```diff
@@ -25,14 +25,17 @@ def get_current_datetime(self, _: dict) -> BaseArtifact:
         "schema": Schema({
             Literal(
                 "relative_date_string",
-                description='Relative date in English. For example, "now EST", "20 minutes ago", or "yesterday at 2pm"'
+                description='Relative date in English. For example, "now EST", "20 minutes ago", '
+                            '"in 2 days", "3 months, 1 week and 1 day ago", or "yesterday at 2pm"'
             ): str
         })
     })
     def get_relative_datetime(self, params: dict) -> BaseArtifact:
         try:
             date_string = params["values"]["relative_date_string"]
             relative_datetime = parse(date_string)
+            if not relative_datetime:
+                return ErrorArtifact("invalid date string")
 
             return TextArtifact(str(relative_datetime))
```

Comment:
Let's always have an `if-else` structure for all returns. General rule should be "no early returns" as they make code harder to follow and can result in subtle bugs.